### DISCUSSION TITLE
react select selected item append separately

### DISCUSSION THREAD

**Discussion by sajumani** [JedWatson/react-select#5337] [Q&A]
![sample](https://user-images.githubusercontent.com/7368727/189472693-9ff1fd74-4fd8-4d4b-94d1-04180d743058.jpg)

Am using react select in all my project, I could see one limitation

When were we using with long list, Or in small area selected items are appending in same select box.

Am expecting to append that in external container

**Comment by manjushsh** [comment]
What is the limitation here?

**Comment by sajumani** [comment]
I wanted to append, selected item to external container, Now its appending to same  .. so it looks agley for long selection
